src/software/logger/logger.h

```diff
@@ -80,6 +86,12 @@ class LoggerSingleton
         // arg. Note: log locations are defaulted to the bazel-out folder due to Bazel's
         // hermetic build principles
 
+        // if log dir doesn't exist, create it
+        if (!std::filesystem::exists(runtime_dir))
+        {
+            std::filesystem::create_directory(runtime_dir);
```

Comment:
```suggestion
            std::filesystem::create_directories(runtime_dir);
```
I hope this fixes your CI issues for this PR